image_width: 640
image_height: 480
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [500.4771, 0.0, 316.925, 0.0, 499.5254, 235.1705, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.1841,-0.6296,-0.0022,-0.0040,0.8279]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [500.4771, 0.0, 316.925, 0.0, 0.0, 499.5254, 235.1705, 0.0, 0.0, 0.0, 1.0, 0.0]